src/book/03-guides/04-general/01-git.mdx

```diff
@@ -64,6 +66,28 @@ If your code is in a different repository and it uses C++ or protobuf files, use
 
 If you push your changes to GitHub and get a message that the build failed due to formatting issues, try running the relevant format commands, commit any changes, and push to GitHub again. This will trigger a new build, which will check the formatting again. You won't be able to merge the PR until the build checks pass.
 
+## Reviewing Code
+
+To review code on GitHub open up a PR and open the Files Changed tab. This will give you the diff of the PR. You can click on the lines of code in the diff to add a comment and/or a suggestion to those lines. You can either leave a single comment or start a review. In a review all comments you make will have pending written on them to say that they have not been posted yet. Please make each comment non-ambiguous and clearly state what your concern is.
```

Comment:
```suggestion
To review code on GitHub, go to the relevant PR page and open the **Files Changed** tab. This will show the <abbr title="The difference between the current files and the changes in the PR">diff</abbr> of the PR.

You can add <abbr title="General remarks or questions">comments</comments> and/or <abbr title="Specific code changes that can be added automatically if accepted by the PR author">suggestions</abbr> to one or more lines in the diff. For a single line, move your mouse cursor over the line and click the plus button that appears next to the line number. For multiple lines, click and drag the plus button down, starting on the first line. A Markdown editor will pop up to write your comment or suggestion. To make a code suggestion, click the button that looks like a file icon with **±** inside it, and then make your changes to the code snippet that is inserted. Click the **Preview** tab to preview your changes. When done, you can either leave a single comment or start a review. In a review, all comments you make will be pending until you finish your review, when they will all be posted at once.

Make each comment non-ambiguous and clearly state what your concern is. Be kind and make sure to word your comments constructively. Prefer "we" over "you" when making suggestions for what to do.
```